clientd3d/client.def

```diff
@@ -189,6 +189,8 @@ EXPORTS
 	GetRoomObjectById
 	FindVisibleObjectById
 	GetPlayerInfo
+	SetActiveGroup
+	GetActiveGroup
```

Comment:
Just "group" is too generic.  Can you rename to "StatGroup" / stat_group everywhere?